iroh-relay/src/server.rs

```diff
@@ -42,8 +42,7 @@ use crate::{
     quic::server::{QuicServer, ServerHandle as QuicServerHandle},
 };
 
-pub(crate) mod actor;
-pub(crate) mod client_conn;
+mod client;
```

Comment:
We now have `mod client` and `mod clients`, which seems slightly confusing.

Reply:
a little, but `client_conn` was a weird name as well imo